app/assets/javascripts/discourse/tests/integration/components/emoji-uploader-test.js

```diff
@@ -29,16 +33,30 @@ discourseModule("Integration | Component | emoji-uploader", function (hooks) {
     });
 
     pretender.post("/admin/customize/emojis.json", () => {
-      return [
-        200,
-        { "Content-Type": "application/json" },
-        {
-          group: "default",
-          name: "test",
-          url:
-            "//upload.s3.dualstack.us-east-2.amazonaws.com/original/1X/123.png",
-        },
-      ];
+      if (requestNumber === 1) {
```

Comment:
This feels so icky, I long for ruby mocha for requests when I have to write tests like this...

Reply:
It was a mistake to move pretender calls to `beforeEach`. I want to fix that as soon as we're off the legacy ember env…